packages/suite/test/health/urls.test.ts

```diff
@@ -1,6 +1,6 @@
 import fetch from 'node-fetch';
 
-import * as suiteUrls from '@suite/constants/urls';
+import * as suiteUrls from '@suite/constants/suite/urls';
```

Comment:
old import

Reply:
hmm.. this would require adding module names mapping to `jest.config.health` is that what we want?

Reply:
since they already are defined in `jest.config.js` you could just use `require()` to load the config and copy the definitions from it.

Reply:
rekt